packages/logging/src/setup-logger-and-telemetry.ts

```diff
@@ -113,6 +113,7 @@ export function setupLoggerAndTelemetry(
 
   bus.on('mongosh:connect', function (args: ConnectEvent) {
     const connectionUri = redactURICredentials(args.uri);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Really wish these rules would understand underscore-prefixed names or at least some kind of pattern for avoiding false positives :slightly_smiling_face: